src/components/KTableView/KTableView.vue

```diff
@@ -1015,23 +1029,24 @@ watch(() => headers, (newVal: readonly Header[]) => {
 
 const isColumnSortable = (column: TableViewHeader<ColumnKey>): boolean => !column.hideLabel && !!column.sortable && column.key !== TableViewHeaderKeys.BULK_ACTIONS && column.key !== TableViewHeaderKeys.ACTIONS
 
-const sortClickHandler = (header: TableViewHeader<ColumnKey>): void => {
-  const { key } = header
-
-  if (sortColumnKey.value) {
-    if (key === sortColumnKey.value) {
+const sortClickHandler = (columnKey: ColumnKey | ''): void => {
```

Comment:
it's hard to tell what's going on in this method now with all of the changes. Can you please add some comments to help with reviewing?